module/actuation/Servos/src/Servos.hpp

```diff
@@ -51,26 +52,30 @@ namespace module::actuation {
         // TODO(ysims): add capability to be Done when the servo reaches the target position
         template <typename Servo, ServoID::Value ID>
         void add_servo_provider() {
-            on<Provide<Servo>, Every<90, Per<std::chrono::seconds>>, Priority::HIGH>().then(
-                [this](const Servo& servo, const RunReason& run_reason) {
-                    if (run_reason == RunReason::NEW_TASK) {
-                        if (log_level <= DEBUG) {
-                            emit(graph("Servo " + std::to_string(ID) + " (Position, Gain, Torque Enabled): ",
-                                       servo.command.position,
-                                       servo.command.state.gain,
-                                       servo.command.state.torque));
-                        }
-                        emit(std::make_unique<ServoTarget>(servo.command.time,
-                                                           ID,
-                                                           servo.command.position,
-                                                           servo.command.state.gain,
-                                                           servo.command.state.torque));
+            on<Provide<Servo>, Priority::HIGH>().then([this](const Servo& servo, const RunReason& run_reason) {
+                if (run_reason == RunReason::NEW_TASK) {
```

Comment:
I think the logic in this function can be simplified. Just emit on new task, but also if time has passed and waiting in time then emit done, otherwise emit wait

Reply:
this is what it there currently

Reply:
I think I was reviewing before that commit happened